src/Components/creatorPage/AuthorInfoCard.js

```diff
@@ -0,0 +1,158 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Card from '@mui/material/Card';
```

Comment:
Just for concision's sake, can you consolidate all the imports from @mui/material into one line rather than importing each component from their specific file?